src/core/sprites/webgl/SpriteRenderer.js

```diff
@@ -138,6 +136,8 @@ export default class SpriteRenderer extends ObjectRenderer
 
         this.vao = this.vaos[0];
         this.currentBlendMode = 99999;
+
+        this.boundTextures = new Array(this.MAX_TEXTURES);
```

Comment:
Unfortunatly we cant, we only know MAX_TEXURES once we have create the context as we need to read that value from it.

I could add the variable though ```this.boundTextures = null```?